php/stats.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 include_once 'locale.php';
+include_once 'config.php';
```

Comment:
We use `$OF_DEMO_UID` so we shouldn't rely on a transitive include from `locale.php`.